src/css/components/billboard.scss

```diff
@@ -39,3 +39,12 @@
     line-height: 2.4375rem;
   }
 }
+
+// testing
+
+.section-card {
+  border: 1px solid #bababa; // having trouble referencing $color-lightgray;
```

Comment:
You probably need to import the WDS defaults file: `@import "../../node_modules/uswds/src/stylesheets/core/_defaults.scss";`